update/common/scripts/5.4/cleanuntranslatablerelations.php

```diff
@@ -53,9 +53,16 @@
  */
 function parseRelationListIds( $relationListXml )
 {
+    if ( empty( trim( $relationListXml ) ) )
+        return array();
+
     $xml = simplexml_load_string( $relationListXml );
     $list = $xml->{'relation-list'};
 
+    if ( ! $list instanceof SimpleXMLElement ) {
```

Comment:
lol, using '! $x instanceof $y' is one of the things I always do and cross my fingers hoping it will work :-D so parenthesis are fine with me.
If not using parenthesis, please at least never use '!$x instaceof $y' and keep the initial space, as otherwise brain goes shortcircuit